fem/ceed/interface/operator.hpp

```diff
@@ -53,6 +52,7 @@ class Operator : public mfem::Operator
 
 #ifdef MFEM_USE_CEED
    CeedOperator& GetCeedOperator() { return oper; }
+   operator CeedOperator() const { return oper; }
```

Comment:
Is this implicit cast needed?

Reply:
Done in [6fe7e0f](https://github.com/mfem/mfem/pull/2952/commits/6fe7e0f8b8148b7084c3e118653eade6b77c2f7e).